Mod3_Q8.cpp

```diff
@@ -0,0 +1,31 @@
+#include <iostream>
+
+
+int main()
+{
+    int x;
+    int y;
+    int z;
+    int a;
+    
+    std::cout << "Enter a value for x ";
+    std::cin >> x;
+    std::cout << "Enter a value for y ";
+    std::cin >> y;
+    
+    z = x + y;
+
+    std::cout << z;
+    
+    a = z%2;

```

Comment:
You could possibly add this as part of the condition on line 22 rather than having a separate variable for it. Only because it is not used anywhere else.